src/main/java/fi/hsl/common/passengercount/PassengerCountParser.java

```diff
@@ -29,6 +29,17 @@ public static PassengerCountParser newInstance() {
 
     @NotNull
     public Optional<PassengerCount.Payload> parsePayload(@NotNull ApcJson json) {
+        final PassengerCount.Payload payload = parsePassengerCountPayload(json);
+        
+        if (payload.hasInvalid()) {
```

Comment:
I'd prefer "isInvalid" instead of "hasInvalid" here, semantically makes more sense in this context